sql/api/src/main/scala/org/apache/spark/sql/catalyst/trees/QueryContexts.scala

```diff
@@ -28,19 +28,20 @@ case class SQLQueryContext(
     sqlText: Option[String],
     originObjectType: Option[String],
     originObjectName: Option[String]) extends QueryContext {
+  override val contextType = QueryContextType.SQL
 
-  override val objectType = originObjectType.getOrElse("")
```

Comment:
why remove `override`?

Reply:
this chnage accidentally remained here from a different implementation. I will remove it.